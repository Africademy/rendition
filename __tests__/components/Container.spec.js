/* globals expect, test */
import React from 'react'
import renderer from 'react-test-renderer'
import {
  Container,
  Provider
} from '../../dist'

test('Container renders correctly', () => {
  const component = renderer.create(
    <Provider>
      <Container>Container</Container>
    </Provider>
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
