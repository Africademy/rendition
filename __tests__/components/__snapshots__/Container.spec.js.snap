// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders correctly 1`] = `
.c1 {
  font-family: 'Nunito',Arial,sans-serif;
  font-size: 14px;
  line-height: 1.5;
  box-sizing: border-box;
  -webkit-text-size-adjust: 100%;
  -ms-text-size-adjust: 100%;
  -moz-osx-font-smoothing: grayscale;
  -webkit-font-smoothing: antialiased;
}

.c0 {
  font-family: 'Nunito',Arial,sans-serif;
}

.c0 h1,
.c0 h2,
.c0 h3,
.c0 h4,
.c0 h5,
.c0 h6,
.c0 label,
.c0 button {
  font-family: CircularStd,Arial,sans-serif;
}

.c3 {
  max-width: calc(32em - 0em);
}

.c2 {
  margin-left: auto;
  margin-right: auto;
  padding-left: 16px;
  padding-right: 16px;
}

@media screen and (min-width:32em) {
  .c3 {
    max-width: calc(48em - 4em);
  }
}

@media screen and (min-width:48em) {
  .c3 {
    max-width: calc(64em - 8em);
  }
}

@media screen and (min-width:64em) {
  .c3 {
    max-width: calc(80em - 16em);
  }
}

<div
  className="c0 c1"
>
  <div
    className="c2 c3"
  >
    Container
  </div>
</div>
`;
